src/components/tutorial/FeedbackButton.astro

```diff
@@ -140,6 +159,13 @@ const instanceId = Math.round(Math.random() * 10e9).toString(36);
 			openBtn.addEventListener('click', openModal);
 			closeBtn.addEventListener('click', closeModal);
 
+			// Pre-fills the GitHub issue with any message a site visitor has already typed in the message box
+			githubButton?.addEventListener('click', () => {
+				if (form.text.value) 
+					githubButton.href = `https://github.com/withastro/docs/issues/new?body=${form.text.value}`
```

Comment:
Should the form input be encoded to escape characters like `&` and `=` that might appear in the body? `?body=death & taxes` becomes `?body=death%20%26%20taxes`

```suggestion
					githubButton.href = `https://github.com/withastro/docs/issues/new?body=${encodeURIComponent(form.text.value)}`
```

Reply:
That's true! Thanks for the suggestion!